guides/source/action_text_overview.md

```diff
@@ -8,149 +8,141 @@ rich text content.
 end
 ```
 
-or add rich text field while creating a new model using:
-
-```bash
-$ bin/rails generate model Message content:rich_text
-```
-
-NOTE: you don't need to add a `content` field to your `messages` table.
-
-Then use [`rich_text_area`][] to refer to this field in the form for the model:
+NOTE: The `content` column will not need to be added to your Blog table. `has_rich_text` helps to associate it with the `action_text_rich_texts` table that has been created and links it back to your model. You also may choose to name the attribute to be something different to `content`.
+Once you have added the `has_rich_text` class method to the model, you can then update your views to make use of the rich text editor (Trix) for that field. In order to do so, use a  `rich_text_area` for the form field.
 
-```erb
-<%# app/views/messages/_form.html.erb %>
-<%= form_with model: message do |form| %>
+```html+erb
+<%# app/views/blogs/_form.html.erb %>
+<%= form_with model: blog do |form| %>
   <div class="field">
     <%= form.label :content %>
     <%= form.rich_text_area :content %>
   </div>
 <% end %>
 ```
 
-To accept the rich text content, all you have to do is permit the referenced attribute:
+This will display a Trix editor that provides the functionality to create and update your rich text accordingly. Styling updates for the editor can be made in `actiontext.css`.
+
+Finally, in order to ensure that you can accept updates from the editor, you will need to permit the the referenced attribute as a parameter in the relevant controller:
 
 ```ruby
-class MessagesController < ApplicationController
+class BlogsController < ApplicationController
   def create
-    message = Message.create! params.require(:message).permit(:title, :content)
-    redirect_to message
+    blog = Blog.create! params.require(:blog).permit(:title, :content)
+    redirect_to blog
   end
 end
 ```
 
-NOTE: Since Action Text relies on polymorphic associations, and [polymorphic associations](./association_basics.html#polymorphic-associations) rely on storing class names in the database, that data must remain synchronized with the class name used by the Ruby code. When renaming classes that use `has_rich_text`, make sure to also update the class names in the `action_text_rich_texts.record_type` polymorphic type column of the corresponding rows.
+If the need arises to rename classes that utilize `has_rich_text`, you will also need to update the polymorphic type column `record_type` in the  `action_text_rich_texts` table for the respective rows.
 
-[`rich_text_area`]: https://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-rich_text_area
+Since Action Text depends on polymorphic associations, which, in turn, involve storing class names in the database, it's crucial to keep the data in sync with the class names used in your Ruby code. This synchronization is essential to maintain consistency between the stored data and the class references in your codebase.
```

Comment:
Should we link to the polymorphic associations section of the Active Record Associations guide? https://guides.rubyonrails.org/association_basics.html#polymorphic-associations
It's a phrase that made me pause and I'm sure other beginners would too.

Reply:
Or we can create a little info box to do that: "With polymorphic associations, a model can belong to more than one other model, on a single association. Read more on the Active Record Associations guide." https://guides.rubyonrails.org/association_basics.html#polymorphic-associations

Reply:
Done ✅ I've added it as an info box in the installation section where I first mention the term polymorphic relationship.

Reply:
@carlosantoniodasilva I'm going to give this one more thought as it will take some more co-ordination and won't be ready before the blog announcement. I will follow up with a PR containing this and some others that need consideration.